test/SlowTests/Issues/RavenDB-19967.cs

```diff
@@ -72,7 +75,7 @@ public async Task DismissTombstoneNotification()
             }
         }
 
-        [Fact]
+        [RavenFact(RavenTestCategory.Replication | RavenTestCategory.Indexes)]
```

Comment:
and here..
worth go over the all test changed in this PR and verify they are categorized properly